src/components/MainRouter/MainRouter.test.tsx

```diff
@@ -47,10 +49,12 @@ describe("Authenticated User", () => {
 
   [
     { initialPath: "/home", expectedPage: "home" },
+    { initialPath: "/offers", expectedPage: "offers" },
     { initialPath: "/login", expectedPage: "login" },
     { initialPath: "/", expectedPage: "home" },
     { initialPath: "/bad-route", expectedPage: "home" },
     { initialPath: "/home/bad-route", expectedPage: "home" },
+    { initialPath: "/offers/bad-route", expectedPage: "home" },
```

Comment:
Note that since there isn't any nested routing atm **/offers/bad-route** isn't handled by a specific Offers Router that would redirect it to **/offers** rather it is considered as a whole as a bad route (as of this moment) and thus we default the routing to home.